packages/gafl-webapp-service/src/pages/licence-details/licence-start-time/__tests__/route.spec.js

```diff
@@ -137,3 +144,79 @@ describe('licenceStartTimeRoute > onPostAuth', () => {
     })
   })
 })
+
+describe('getData', () => {
+  const getData = pageRoute.mock.calls[1][4]
+  const mockTransactionCacheGet = jest.fn()
+  const mockRequest = {
+    cache: () => ({
+      helpers: {
+        transaction: {
+          getCurrentPermission: mockTransactionCacheGet
+        }
+      }
+    }),
+    locale: 'en-gb'
+  }
+
+  it('startDateStr is result of moment().format()', async () => {
+    const formatSymbol = Symbol('formatResult')
+    moment.mockImplementation(
+      getMomentMockImpl({
+        format: () => formatSymbol
+      })
+    )
+    mockTransactionCacheGet.mockImplementationOnce(() => ({ licenceStartDate: '2020-01-01' }))
+    const { startDateStr } = await getData({
+      cache: () => ({
+        helpers: {
+          transaction: {
+            getCurrentPermission: mockTransactionCacheGet
+          }
+        }
+      }),
+      locale: 'anywhere'
+    })
+    expect(startDateStr).toEqual(formatSymbol)
+  })
+
+  it.each([['2020-07-06'], ['2020-01-01'], ['2022-07-09']])('permission licence start date is passed to moment', async licenceStartDate => {
+    mockTransactionCacheGet.mockImplementationOnce(() => ({ licenceStartDate }))
+    moment.mockImplementation(getMomentMockImpl())
+    await getData(mockRequest)
+    expect(moment).toHaveBeenCalledWith(licenceStartDate, 'YYYY-MM-DD', 'en-gb')
+  })
+
+  it('cacheDateFormat being what is expected', async () => {
+    mockTransactionCacheGet.mockImplementationOnce(() => ({ licenceStartDate: '2020-01-01' }))
+    moment.mockImplementation(getMomentMockImpl())
+    await getData(mockRequest)
+    expect(moment).toHaveBeenCalledWith('2020-01-01', 'YYYY-MM-DD', 'en-gb')
+  })
```

Comment:
cacheDateFormat is imported from `date-and-time-display.js`, so I think you need to mock it (again, as a Symbol), then (as you don't care what the first and third arguments are) assert with:
```
expect(moment).toHaveBeenCalledWith(expect.any(String), mockCacheDateFormat, expect.any(String))
```
Then you need a test to check that request.locale is passed to moment - use the same approach as you have for this and the previous test, mocking `request.locale` as a `Symbol` again.